### ISSUE TITLE
Optionally connect to running Chrome instance

### ISSUE THREAD

**Issue by soulchild** [sindresorhus/capture-website#32]
Thanks for creating this very useful wrapper around puppeteer, Sindre!

Currently, it launches a Chrome instance for every capture via `puppeteer.launch()`. Would it make sense, if we added an option to let it connect to a running Chrome instance via [puppeteer.connect()](https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md#puppeteerconnectoptions)? I could take a look at implementing this, if there are no obvious (or not so obvious) arguments against it. 😄

**Comment by stefpb** [comment]
Yes, i would be nice, because `puppeter.connect()` is faster: https://stackoverflow.com/a/52434730/11830242

**Comment by AlejandroAkbal** [comment]
Yes please, I have tons of zombie processes because of this

**Comment by soulchild** [comment]
I decided to use the puppeteer library directly, because for my use-case this library was just a small wrapper around it anyways. 

I had to deal with zombie processes as well (in a Docker/K8s environment) and the solution was to use [tini](https://github.com/krallin/tini) as my Docker entrypoint which takes care of correctly reaping sub-process zombies.

**Comment by AlejandroAkbal** [comment]
Very interesting, I might have to do the same for my https://github.com/AlejandroAkbal/Screenshot-API project

Thanks for the info @soulchild 🙏

**Comment by sindresorhus** [comment]
Going to pass on this.
